docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/Recommendation/MatrixFactorizationWithOptions.cs

```diff
@@ -10,44 +10,55 @@ namespace Samples.Dynamic.Trainers.Recommendation
     public static class MatrixFactorizationWithOptions
     {
         
-        // This example requires installation of additional nuget package <a href="https://www.nuget.org/packages/Microsoft.ML.Recommender/">Microsoft.ML.Recommender</a>.
-        // In this example we will create in-memory data and then use it to train 
-        // a matrix factorization model with non-default parameters. Afterward, quality metrics are reported.
+        // This example requires installation of additional nuget package at
+        // for Microsoft.ML.Recommender at
+        // https://www.nuget.org/packages/Microsoft.ML.Recommender/
+        // In this example we will create in-memory data and then use it to train
+        // a matrix factorization model with default parameters. Afterward, quality
+        // metrics are reported.
         public static void Example()
         {
-            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
-            // as a catalog of available operations and as the source of randomness.
-            // Setting the seed to a fixed number in this example to make outputs deterministic.
+            // Create a new context for ML.NET operations. It can be used for
+            // exception tracking and logging, as a catalog of available operations
+            // and as the source of randomness. Setting the seed to a fixed number
+            // in this example to make outputs deterministic.
             var mlContext = new MLContext(seed: 0);
 
             // Create a list of training data points.
             var dataPoints = GenerateMatrix();
 
-            // Convert the list of data points to an IDataView object, which is consumable by ML.NET API.
+            // Convert the list of data points to an IDataView object, which is
+            // consumable by ML.NET API.
             var trainingData = mlContext.Data.LoadFromEnumerable(dataPoints);
 
             // Define trainer options.
             var options = new MatrixFactorizationTrainer.Options
             {
-				// Specify IDataView colum which stores matrix column indexes. 
-				MatrixColumnIndexColumnName = nameof(MatrixElement.MatrixColumnIndex),
-				// Specify IDataView colum which stores matrix row indexes. 
+                // Specify IDataView colum which stores matrix column indexes. 
+                MatrixColumnIndexColumnName = nameof(MatrixElement.MatrixColumnIndex
+                    ),
+
+                // Specify IDataView colum which stores matrix row indexes. 
                 MatrixRowIndexColumnName = nameof(MatrixElement.MatrixRowIndex),
-				// Specify IDataView colum which stores matrix elements' values. 
+                // Specify IDataView colum which stores matrix elements' values. 
```

Comment:
Is this indenation mismatch caused by tabs? If yes, did you also fix the `*.tt` file?